README.md

```diff
@@ -72,6 +73,7 @@ export RESOURCE_GROUP_NAME=give_your_name_citus_test_automation_r_g
 # export REGION=eastus2
 
 # open and modify the instance types/discs as you wish
+# for valgrind tests, set `numberOfWorkers` to `0` as we only need a single machine cluster 
```

Comment:
I think we can be even more explicit here to clarify why we only need a single machine cluster.

Maybe:

`for valgrind tests, set `numberOfWorkers` to `0` as we only need a single machine cluster. This is because we use our regression test structure for valgrind and it creates a local cluster.

Reply:
Upated the comment